### ISSUE TITLE
On a Service peer, allow the Service opperator to restrict by peerid who can use it

### ISSUE THREAD

**Issue by andrewxhill** [textileio/go-threads#316]
Goal: Allow a user to setup a Threadsd that _only accepts follow requests from a limited set of peers_.

One solution might be to enable a service peers whitelist in the config. That whitelist could just contain the peerids of anyone you are willing to accept follow requests from. 

Another might be the ACL, but I'm not sure yet how that would work in this situation.

**Comment by andrewxhill** [comment]
something to consider here is https://github.com/textileio/go-threads/issues/233

i still get errors such as,

```
2020-05-01T09:50:09.731-0700	FATAL	db	db/collection.go:462	invalid instance: doesn't have an _id attribute
exit status 1
~/Textile/thre
``` 

I can't tell if that falls into the case of the closed ticket above. But, if people start running this in systems where groups of users/apps connect, they may not want the Fatal fails so easily.

**Comment by andrewxhill** [comment]
cc @jsign in case I misinterpreted 233 and that fatal actually shouldn't be fatal

**Comment by jsign** [comment]
Yes, that shouldn't be fatal since there isn't anything corrupt or similar... just an expected error.

**Comment by andrewxhill** [comment]
https://github.com/textileio/go-threads/pull/331
